docs/Makefile

```diff
@@ -0,0 +1,23 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Need to delete the root Makefile, otherwise it looks like we can build the doc from the root Makefile:

```
Application error:
Cannot find source directory (/Users/tdeboisfosse/projects/opensource/substra-documentation/src)
make: *** [html] Error 2
```

Reply:
+ the root conf.py I guess

Reply:
How do you get this error?

Reply:
if I try to call a `make` command from the root directory
now we have to go to the `docs` directory to run make